Source/CarthageKit/BinaryJSONError.swift

```diff
@@ -21,10 +20,13 @@ extension BinaryJSONError: CustomStringConvertible {
 		switch self {
 		case let .invalidJSON(error):
 			return "invalid JSON: \(error)"
+
```

Comment:
You prefer

```Swift
                switch self {
		case let .invalidJSON(error):
			return "invalid JSON: \(error)"
		case let .invalidVersion(error):
			return "unable to parse semantic version: \(error)"
		case let .invalidURL(string):
			return "invalid URL: \(string)"
		case let .nonHTTPSURL(url):
			return "specified URL '\(url)' must be HTTPS"
		}
```
to
```Swift
                switch self {
		case let .invalidJSON(error):
			return "invalid JSON: \(error)"

		case let .invalidVersion(error):
			return "unable to parse semantic version: \(error)"

		case let .invalidURL(string):
			return "invalid URL: \(string)"

		case let .nonHTTPSURL(url):
			return "specified URL '\(url)' must be HTTPS"
		}
```
?

Why? The latter is clearly more readable to me ... not for you, too?